packages/@dcl/ecs/package.json

```diff
@@ -25,6 +25,9 @@
     "build": "tsc -p tsconfig.json && tsc -p tsconfig.cjs.json",
     "start": "tsc -p tsconfig.json --watch"
   },
+  "dependencies": {
```

Comment:
The circular dependency was providing `protobufjs` to `@dcl/ecs`, which still needs it upon the very first build process to compile the relevant `.proto` files.